### DISCUSSION TITLE
Latest BSL to GNU release tracker

### DISCUSSION THREAD

**Discussion by SayanoAI** [directus/directus#23979] [Feature Requests]
### Summary

Based on your BSL license page, it seems like the BSL license gets converted to GNU after 3 years.
Is there anywhere we can easily view which of the latest version got changed to GNU?
![image](https://github.com/user-attachments/assets/4eac3f7e-bf45-49f3-bed1-a341f909ebde)

### Basic Example

e.g. Latest GNU version: 10.42

### Motivation

This will help people more easily use the latest GNU version without taking up time from your sales team.

### Detailed Design

Keep a list of release versions and dates since BSL implementation and fetch the latest version that was released before: `Date.now() - 3 years` or `<date that BSL was first implemented>`.

### Must Have:
- Latest GNU version

### Should Have:
- N/A

### Could Have:
- Date next version gets GNUed

### Won't Have:
- N/A

### Drawbacks

It could impact your revenue if people are fine with using older versions.

### Alternatives

Someone could use git history to track it I suppose. Easily to dispute if it doesn't come from an official source.

### Adoption Strategy

The latest GNU version could be added to your website or doc page for developers to more easily find that information.
No migration is needed since users can just do `npm i directus@[insert latest GNU version]`.

**Comment by rijkvanzanten** [comment]
All releases are listed here incl. their release date: https://github.com/directus/directus/releases 👍🏻
